Index: Assets/Plugins/Zenject/OptionalExtras/Signals/Zenject-Signals.csproj.meta
===================================================================
diff --git a/Assets/Plugins/Zenject/OptionalExtras/Signals/Zenject-Signals.csproj.meta b/Assets/Plugins/Zenject/OptionalExtras/Signals/Zenject-Signals.csproj.meta
deleted file mode 100644
--- a/Assets/Plugins/Zenject/OptionalExtras/Signals/Zenject-Signals.csproj.meta	
+++ /dev/null	
@@ -1,7 +0,0 @@
-fileFormatVersion: 2
-guid: aebd7ef925dd4fe4690c3238e879f914
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
Index: Assets/Plugins/Zenject/Source/Zenject.csproj.meta
===================================================================
diff --git a/Assets/Plugins/Zenject/Source/Zenject.csproj.meta b/Assets/Plugins/Zenject/Source/Zenject.csproj.meta
deleted file mode 100644
--- a/Assets/Plugins/Zenject/Source/Zenject.csproj.meta	
+++ /dev/null	
@@ -1,7 +0,0 @@
-fileFormatVersion: 2
-guid: 8645273165cb7f54290d6eaa1e10ab37
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
Index: Assets/Plugins/Zenject/OptionalExtras/ReflectionBaking/Zenject-ReflectionBaking.csproj.meta
===================================================================
diff --git a/Assets/Plugins/Zenject/OptionalExtras/ReflectionBaking/Zenject-ReflectionBaking.csproj.meta b/Assets/Plugins/Zenject/OptionalExtras/ReflectionBaking/Zenject-ReflectionBaking.csproj.meta
deleted file mode 100644
--- a/Assets/Plugins/Zenject/OptionalExtras/ReflectionBaking/Zenject-ReflectionBaking.csproj.meta	
+++ /dev/null	
@@ -1,7 +0,0 @@
-fileFormatVersion: 2
-guid: a6d77ad9009029a4bb0824ee8e302359
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
Index: Assets/Spine/spine-csharp/spine-csharp.sln.meta
===================================================================
diff --git a/Assets/Spine/spine-csharp/spine-csharp.sln.meta b/Assets/Spine/spine-csharp/spine-csharp.sln.meta
deleted file mode 100644
--- a/Assets/Spine/spine-csharp/spine-csharp.sln.meta	
+++ /dev/null	
@@ -1,7 +0,0 @@
-fileFormatVersion: 2
-guid: 4c9ac72997a2f284a89d08ba9704566e
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
Index: Assets/Spine/spine-csharp/spine-csharp.csproj.meta
===================================================================
diff --git a/Assets/Spine/spine-csharp/spine-csharp.csproj.meta b/Assets/Spine/spine-csharp/spine-csharp.csproj.meta
deleted file mode 100644
--- a/Assets/Spine/spine-csharp/spine-csharp.csproj.meta	
+++ /dev/null	
@@ -1,7 +0,0 @@
-fileFormatVersion: 2
-guid: e3106e30cda175b49bb18e20c47903e6
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
